libs/barista-components/stacked-series-chart/src/_stacked-series-chart-theme.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../core/src/style/colors';
+
+.dt-theme-dark :host {
+  .dt-stacked-series-chart-value-axis {
+    background-color: $gray-860 !important;
+  }
+}
+
+.dt-theme-dark .dt-card :host {
+  .dt-stacked-series-chart-value-axis {
+    background-color: #505050 !important;
```

Comment:
Please avoid using !important, if you have rules that have the same specificity you can duplicate a class for instance 
```
.dt-stacked-series-chart-value-axis.dt-stacked-series-chart-value-axis
```

Reply:
Thanks for the input, I'll apply this change